lektor/devserver.py

```diff
@@ -99,7 +94,7 @@ def browse():
         webbrowser.open("http://%s:%s" % addr)
 
     t = threading.Thread(target=browse)
-    t.setDaemon(True)
+    t.daemon = True
```

Comment:
As of python 3.3, one can pass a `daemon` kwarg to `Thread.__init__`. E.g.:
```py
t = threading.Thread(target=browse, daemon=True)
t.start()
```

or even
```py
threading.Thread(target=browse, daemon=True).start()
```

Reply:
Ah, good to know. In the other case the `BackgroundBuilder` class should probably already do that itself.